client/driver/lxc.go

```diff
@@ -0,0 +1,498 @@
+//+build linux
+
+package driver
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/hashicorp/nomad/client/config"
+	"github.com/hashicorp/nomad/client/fingerprint"
+	"github.com/hashicorp/nomad/client/stats"
+	"github.com/hashicorp/nomad/helper/fields"
+	"github.com/hashicorp/nomad/nomad/structs"
+	"github.com/mitchellh/mapstructure"
+
+	dstructs "github.com/hashicorp/nomad/client/driver/structs"
+	cstructs "github.com/hashicorp/nomad/client/structs"
+	lxc "gopkg.in/lxc/go-lxc.v2"
+)
+
+const (
+	// The option that enables this driver in the Config.Options map.
+	lxcConfigOption = "driver.lxc.enable"
```

Comment:
Fixed the comment.

@diptanu did you get a chance to write any docs? If not I don't mind doing it.